.eslintrc.js

```diff
@@ -19,5 +19,17 @@ module.exports = {
 	],
 	extends: [
 		"plugin:@wordpress/eslint-plugin/recommended",
-	]
+	],
+	settings: {
+		'import/resolver': {
+			'node': {
+				'moduleDirectory': ['node_modules', 'gutenberg', 'gutenberg/node_modules']
```

Comment:
Would it make sense to add `jetpack` here too?

Reply:
Currently metro [doesn't check](https://github.com/wordpress-mobile/gutenberg-mobile/blob/a53c06da987bda6de57b53aede55065c989c8413/metro.config.js) the `node_modules` folder in there, I don't think it would make sense to add it.

Also I removed `gutenberg` from there as it was not working as I expected. Building `gutenberg` packages before linting instead did the trick because it symlinked the `@wordpress/` packages inside `gutenberg/node_modules`.

Reply:
Found the reason: https://github.com/WordPress/gutenberg/pull/22088